components/tools/OmeroPy/src/omero/util/script_utils.py

```diff
@@ -1234,3 +1230,34 @@ def findROIByImage(roiService, image, namespace):
     for roi in results.rois:
         roiList.append(ROIData(roi))
     return roiList
+
+
+def numpyToImage(plane, minMax, type):
+    """
+    Converts the numpy plane to a PIL Image, converting data type if necessary.
+    @param plane The plane to handle
+    @param minMax the min and the max values for the plane
+    @param type the data type to use for scaling
+    """
+
+    import numpy
```

Comment:
it is certainly not in try/except in many places.
We don't need to do it anymore I think